<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SvgParser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="line_sourcePath">
        <property name="text">
         <string>/home/rainovsky/Projects/Svg2Gcode/tests/image/Bicycle.svg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_sourcePath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="line_targetPath">
        <property name="text">
         <string>/home/rainovsky/Projects/Svg2Gcode/tests/image/Bicycle.nc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_targetPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="cb_logFile">
        <property name="text">
         <string>Вывод в лог. файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_logConsole">
        <property name="text">
         <string>Вывод консоль</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>Обработать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
